editor/src/core/workers/parser-printer/parser-printer-parsing.ts

```diff
@@ -1265,8 +1268,6 @@ function pullOutElementsToParse(nodes: Array<TS.Node>): Either<string, ElementsT
       TS.isJsxFragment(node)
     ) {
       result.push(node)
-    } else {
```

Comment:
removed the else condition here because ternaries can have arbitrary nodes in here. this doesn't feel right.